examples/rollout-canary-with-analysis.yaml

```diff
@@ -0,0 +1,45 @@
+# This example demonstrates a Rollout using the canary update strategy with a customized rollout
```

Comment:
I think we can modify the existing rollout-analysis-step.yaml example to use both AnalysisTemplates and ClusterAnalysisTemplates instead of adding another example